src/components/Authentication/Authentication.vue

```diff
@@ -35,11 +35,12 @@
 </template>
 
 <script>
+import Vue from 'vue';
 import GirderLogin from './Login.vue';
 import GirderRegistration from './Register.vue';
 import { OauthTokenPrefix, OauthTokenSuffix } from '../../rest';
 
-export default {
+export default Vue.extend({
```

Comment:
What's this change for?

Reply:
I make that change in any SFC I touch that doesn't have it when I remember.   It enables VS Code's Vetur plugin to provide template interpolation, so it can do basic type checking and warn about missing variables / typos/ / etc in the template.

It has no impact on the bundle, `vue-loader` feeds the default export to the same `Vue.extend()` function during build.